.github/workflows/ci.yml

```diff
@@ -167,7 +167,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        ruby: ["4.0.0-preview3", "3.4", "3.3", "3.2", "3.1", "3.0"]
```

Comment:
This is not going to work because dockerhub images are not updated, and setup-ruby is not updated.

I would advise just updating the ruby version in the container as a first step.

Reply:
You're right!

Reply:
I've opened an alternative that just updates the version in the container image: https://github.com/rake-compiler/rake-compiler-dock/pull/182